packages/elements/src/datetime-picker/constants.ts

```diff
@@ -0,0 +1,6 @@
+enum DateTimePickerDataName {
+  from = 'from',
+  to = 'to'
+}
+
+export { DateTimePickerDataName };
```

Comment:
Could you re export this in element? So user can import this enum from element itself like slider do.